docs/index.md

```diff
@@ -51,13 +51,12 @@ A guide collaboration between Nintendo Homebrew's Helpers and Staff, from stock
 
     This guide will work on first-generation Switch consoles in all regions on firmware 12.1.0 or below.
 
-    You will need one of the following in order to successfully follow this guide:
+    You will need **one** of the following in order to successfully follow this guide:
 
-    - A USB-A to USB-C cable, and a PC
-    - A USB-OTG cable, a USB-A to USB-C cable, and an Android device
+    - A PC and a USB cable capable of data transfer between your Switch and your PC
+    - An Android device and a USB cable capable of data transfer between your Switch and your Android device
 		- This does not work on every android phone
-    - A USB-C cable, and an Android device with a USB-C port
-    - A Lightning-OTG adapter, a USB-A to USB-C cable, and a jailbroken iOS device
+    - A jailbroken iOS device and a USB cable capable of data transfer between your Switch and your iOS device
```

Comment:
I'd still mention the Lightning-OTG adapter in the iOS part here because injecting a payload with just a cable from iOS to the Switch directly doesn't work.